blackbox/docs/admin/privileges.rst

```diff
@@ -40,6 +40,13 @@ user is allowed to execute ``SELECT``, ``SHOW``, ``REFRESH``, ``COPY TO``,
 and ``SET SESSION`` statements, as well as using the available user defined
 functions, on the object for which the privilege applies.
 
+.. NOTE::
+
+   DQL privileges only be granted for ``SET SESSION`` statements.
```

Comment:
oops. there's a typo here. missing a "can" before "only"

Reply:
Doesn't this just repeat what's already written in the sentence above?